docs/source/transaction_freeze_ledgers.md

```diff
@@ -0,0 +1,15 @@
+# Transaction Freeze Ledgers
+
+#### Why It May Be Needed
+
+- Removing tokens need of mechanism of painless way. Transaction freeze ledgers give a security transition. Transaction Freeze Ledgers disable specific ledgers (system ignore frozen ledgers).
+
+#### Transaction Freeze Ledgers (TFL)
+
+- LEDGERS_FREEZE is transaction for add specific ledgers to list of frozen ledgers.
+
+- Every transaction LEDGERS_FREEZE consists of a list ledgers ids, unique version.
+
+- It's possible to get the whole list of frozen ledgers via transaction GET_FROZEN_LEDGERS.
+
```

Comment:
Please, add that base ledgers POOL, CONFIG and DOMAIN can't be frozen.
And that frozen ledgers can't be unfreeze.